accessman.go

```diff
@@ -223,6 +306,12 @@ func (a *accessMan) newPendingCloseChan(remotePub *btcec.PublicKey) error {
 				// There are no available restricted slots, so
 				// we need to disconnect this peer. We leave
 				// this up to the caller.
+				acsmLog.WarnS(ctxb, "Peer last pending "+
+					"channel closed: ",
+					ErrNoMoreRestrictedAccessSlots,
+					btclog.Hex("peer", []byte(peerMapKey)),
+					"num_restricted", a.numRestricted, "max_restricted", a.cfg.maxRestrictedSlots)
```

Comment:
nit: I think the convention is to have one attribute pair per line.

Reply:
Same further below.

Reply:
> nit: I think the convention is to have one attribute pair per line.

Gotcha, wasn't sure about that, since we rigged the linter to _not_ trigger the line length error for logging statements.